frontend/pages/audit-logs.vue

```diff
@@ -222,20 +439,31 @@ const getTargetLink = (t: AuditLogTargetType, id: string): string => {
   return '#'
 }
 
+const columnControlOverlay = useState<{ toggle: (e: Event) => void }>(`${prefix}.columnControlOverlay`)
```

Comment:
`useState` shouldn't contain things that are unserializable, like functions, since they explode if you try to send that data over SSR